docs/content/en/docs/getting-started/optional/api-server-extra-args.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "API Server Extra Args"
+linkTitle: "API Server Extra Args"
+weight: 10
+aliases:
+    /docs/reference/clusterspec/optional/etcd/
+description: >
+  EKS Anywhere cluster yaml specification API Server Extra Args reference
+---
+
+## API Server Extra Args support (optional)
+
+#### Provider support details
+|                | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:--------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+| **Supported?** |   ✓	    |     ✓      |   	 ✓   |     ✓      |  ✓   |
+
+In order to configure a cluster with api server extra args, you need to configure your cluster by updating the cluster configuration file to include the details below. The feature flag `API_SERVER_EXTRA_ARGS_ENABLED=true` needs to be set.
+
+This is a generic template with some example api server extra args configuration below for reference:
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+   name: my-cluster-name
+spec:
+    ...
+    controlPlaneConfiguration:
+        apiServerExtraArgs:
+            ...
+            service-account-issuer: "https://issuer-url"
+            service-account-jwks-uri: "https://issuer-url/openid/v1/jwks"
+```
+
```

Comment:
It would be nice to show a example here that reflects a real-world customer use case. If this example is useful, I would suggest adding a couple of paragraphs, describing what the example achieves and why someone would do it. In particular, the OIDC use that you describe in the design doc might be a good thing to explain.

Reply:
Updated the template with other API Server options and described some example usecase below